raft/src/test/java/org/apache/kafka/raft/RaftClientTestContext.java

```diff
@@ -1259,6 +1313,27 @@ RaftRequest.Outbound assertSentApiVersionsRequest() {
         return sentRequests.get(0);
     }
 
+    RaftRequest.Outbound assertSentAddVoterRequest(
+        ReplicaKey replicaKey,
+        Endpoints endpoints,
+        boolean expectedAckWhenCommitted
```

Comment:
Let's not give the caller the option to override this. This value should always be false and this method should just check for that explicitly.